Modules/ThirdParty/VNL/src/vxl/CMakeLists.txt

```diff
@@ -155,7 +159,11 @@ set( CMAKE_EXE_LINKER_FLAGS    "${CMAKE_EXE_LINKER_FLAGS} ${VXL_EXTRA_CMAKE_EXE_
 set( CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} ${VXL_EXTRA_CMAKE_MODULE_LINKER_FLAGS}" )
 set( CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} ${VXL_EXTRA_CMAKE_SHARED_LINKER_FLAGS}" )
 
-
+if(MSVC_VERSION GREATER_EQUAL 1920)
+# Force synchronous writes of .pdb files for building VXL on MSVC 1920-1929
+# https://docs.microsoft.com/en-us/cpp/build/reference/fs-force-synchronous-pdb-writes?view=vs-201
+  add_compile_options( "/FS" )
```

Comment:
I believe this will only be needed in debug and relwithdebinfo compilation modes. This should be able to be done with a generator expression. Something like:
```
add_compile_options("$<$<CONFIG:DEBUG>:-/FS>")
add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:-/FS>")
```
This suggestion is due to the caution of performance hit with this option in the linked documentation.

Reply:
Slowdown from serialized access to debug symbol database is moot if debug symbols are not even recorded. I prefer to keep the code simple, as it is now.

Reply:
Ahh, so if debugging not enabled this option should have no effect. Very good then.